register-schemas.R

```diff
@@ -29,7 +29,7 @@ register_schema <- function(file) {
 
 # Register terms ---------------------------------------------------------------
 
-term_files <- dir_ls(here("terms"), regexp = "*\\.json")
+term_files <- dir_ls(here("terms"), regexp = "*\\.json", recurse = TRUE)
```

Comment:
Are we expecting that the directories will be only a single level? If so, `recurse` could be set to an integer, which _might_ save a sliver of time depending on how the fs codebase handles recursion/checks for directories for full recursion. Limiting the level of recursion would add some inflexibility, though.